_posts/2013-08-10-payrolls.markdown

```diff
@@ -133,7 +149,12 @@ layout: sidebar
 | `paid_time_off`           | array of all paid time off (PTO) for which this employee is eligible
 | `name`                    | name of the PTO. This also serves as the unique, immutable identifier for this PTO.
 | `hours`                   | hours taken of this PTO during this pay period.
-
+| `gross_pay`               | employee's gross pay, only available for processed payrolls
+| `net_pay`                 | employee's net pay, only available for prcoessed payrolls
+| `payment_method`          | compensation payment method (direct deposit or check), only available for processed payrolls
```

Comment:
Can we put the right casing and quotes (i.e. "Direct Deposit" or "Check")? This is for developers :)